source/docs/03-metadata.xml

```diff
@@ -1625,6 +1625,43 @@
                               <specDesc key="handList"/>
                            </specList>
                         </p>
+                        <p>
+                           <specList>
+                              <specDesc key="addDesc"/>
+                           </specList>
+                        </p>
```

Comment:
Repeating the specDesc here seems redundant?

Reply:
```suggestion
```